settings.py

```diff
@@ -1415,6 +1415,7 @@
     'corehq.messaging.smsbackends.apposit.models.SQLAppositBackend',
     'corehq.messaging.smsbackends.grapevine.models.SQLGrapevineBackend',
     'corehq.messaging.smsbackends.http.models.SQLHttpBackend',
+    'corehq.messaging.smsbackends.icds.models.SQLICDSBackend',
```

Comment:
Currently there's no way for this to only show up on icds. it comes after the localsettings import. Would it be worthwhile to write code to enable environment specific backends?